integration-cli/daemon/daemon.go

```diff
@@ -180,6 +180,7 @@ func (d *Daemon) getClientConfig() (*clientConfig, error) {
 	if err := sockets.ConfigureTransport(transport, proto, addr); err != nil {
 		return nil, err
 	}
+	transport.DisableKeepAlives = true
```

Comment:
`sockets.ConfigureTransport()` is also used outside of integration-cli;

- [pkg/plugins/client.go#L42](https://github.com/docker/docker/blob/677e14643c79184ca9d8123dd3e91e0444b1cac3/pkg/plugins/client.go#L42)
- [client/client.go#L150](https://github.com/docker/docker/blob/8f9a48ab5a57b33e99a5231a1ea966cd3910373e/client/client.go#L150)
- [cli/command/cli.go#L258](https://github.com/docker/docker/blob/38f766ae0e4b80d452a3825c42a7ac9fee965790/cli/command/cli.go#L258)

Is there ever a need for those to have Keep-Alive enabled? If not, should we make it the default?